lib/tapioca/config.rb

```diff
@@ -21,10 +21,11 @@ def outpath
 
     private_class_method :new
 
-    CONFIG_FILE_PATH = "sorbet/tapioca/config.yml"
-    SORBET_CONFIG = "sorbet/config"
+    TAPIOCA_PATH = T.let("sorbet/tapioca", String)
```

Comment:
What do you think about introducing a `SORBET_PATH` const in using it in all other paths?